index.html

```diff
@@ -9,11 +9,29 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
-</head>
+</head
 
 <body>
     <div class="container">
-   
+        <header>
+            <div class='header-child'>
+                <img src="img/lambda-black.png" alt="">
+            </div>
+            <nav class='header-child'>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Products</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <div class='header-img'>
+                <img src="img/jumbo.jpg" alt="">
+        </div>
```

Comment:
you don't have to put div's around everything; you can just give the img a class